docs/hub/spaces-sdks-docker-aim.md

```diff
@@ -0,0 +1,114 @@
+4. Set the visibility
+
+Once you have created the space, you'll see the `Building` status and once it becomes `Running` your space is ready to go!
+
+### TODO: change screenshot once the Aim template is available
+![Creating an Aim Space ](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-livebook-new-space.png)
+
+Now, when you navigate to the **App** section of your space, you'll be able to access the Aim UI.
+
+## Compare your experiments with Aim on Spaces
+Let's use a quick example of a PyTorch CNN trained on MNIST to demonstrate end-to-end Aim on Spaces deployment. 
+The full example in the [Aim repo examples folder](https://github.com/aimhubio/aim/blob/main/examples/pytorch_track.py).
+
+```python
+from aim import Run
+from aim.pytorch import track_gradients_dists, track_params_dists
+
+# Initialize a new Run
+aim_run = Run()
+...
+items = {'accuracy': acc, 'loss': loss}
+aim_run.track(items, epoch=epoch, context={'subset': 'train'})
+
+# Track weights and gradients distributions
+track_params_dists(model, aim_run)
+track_gradients_dists(model, aim_run)
+```
+
+The experiments tracked bdy Aim are stored in the `.aim` folder. **To display the logs with the Aim UI in your HF space, you need to compress the `.aim` folder to a `tar.gz` file and upload it to your Space either using `git` or the Files and Versions sections of your space.** 
+Here's a bash command for that:
+
+```bash
+tar -czf cnn_mnist_runs_hf.tar.gz .aim
+```
+This command will compress the `.aim` directory to `cnn_mnist_runs_hf.tar.gz`. 
+
+The final step is to edit the compressed file name in the space’s Dockerfile. 
+
+```Dockerfile
+...
+# Note: Change the name from fastspeech_logs.tar.gz to your filename
+COPY fastspeech_logs.tar.gz .
+RUN tar xvzf fastspeech_logs.tar.gz
+```
+That’s it! Now open the App section of your space and the Aim UI is available with your logs.
+Here is what to expect: 
+
+![Aim UI on HF Hub Spaces](https://user-images.githubusercontent.com/67782184/224995208-692afbea-dfe0-4d2a-89ac-4ba552e49a98.png)
+
+Filter your runs using Aim’s Pythonic search. You can write pythonic [queries against](https://aimstack.readthedocs.io/en/latest/using/search.html) EVERYTHING you have tracked - metrics, hyperparams etc.
+In the screenshot below, we query all the `accuracy` metrics where the last tracked value is `≥ 90`.
+
+![image](https://user-images.githubusercontent.com/67782184/224995427-db0d8a2b-1d48-46d1-b519-78a2d4d5d0aa.png)
+
+Check out a [text2speech example ](https://huggingface.co/spaces/aimstack/text2speech/tree/main) on HF Hub Spaces.
+
+**_Note_** that if your logs are in tensorboard format, you can easily convert [them to Aim with one command](https://aimstack.readthedocs.io/en/latest/quick_start/convert_data.html#show-tensorboard-logs-in-aim) and use the many advanced and high-performant training run comparison features available.
+
+## Track your Transformers runs with Aim
```

Comment:
I'm not sure if this should be in the documentation of the Hub, as that's mostly about using the Space. This could maybe be in the transformers docs but also not sure as this is not a in-library callback

Reply:
Yea, our thinking was to make sure there is extra context. We can just link to our Hugging Face integration docs in that case?

Reply:
I think so. Or maybe just have a 2-3 sentence description here and then link to the transformers docs. That will also be easier to maintain